### ISSUE TITLE
Reset score command for player

### ISSUE THREAD

**Issue by 2020karaulov2020** [OciXCrom/RankSystem#9]
You can add command for clear stats. If player enter command like /rs - clear stats for this player.

**Comment by OciXCrom** [comment]
No, I won't be adding such a potentially dangerous command in the main plugin. You can use a 3 lines of code sub-plugin for it if you want to.
Having the plugin set to save by nickname means that ANYONE can enter with your name and RESET everything you have.
